<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>476</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="b_seeothers">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>ZOBACZ INNE POMIARY PULSU</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Twój puls:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>ZAPISZ POMIAR PULSU</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_puls">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>45</pointsize>
    </font>
   </property>
   <property name="text">
    <string>120</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>BPM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_start">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>ROZPOCZNIJ POMIAR PULSU</string>
   </property>
  </widget>
  <widget class="QLabel" name="widget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>165</width>
     <height>168</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="b_save2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>330</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>ZAPISZ POMIAR TEMPERATURY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_seeothers2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>390</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>ZOBACZ INNE POMIARY TEMPERATURY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_start2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>ROZPOCZNIJ POMIAR TEMPERATURY</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Twoja temperatura:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>°C</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_puls_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>45</pointsize>
    </font>
   </property>
   <property name="text">
    <string>39</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
